### ISSUE TITLE
Support for the new Windows Terminal?

### ISSUE THREAD

**Issue by gonza2323** [mabe02/lanterna#612]
Hi,

I understand that the library doesn't support the old cmd.exe terminal. But can it work with the new Windows Terminal? From what I could gather, it supports ANSI escape sequences, and I run lanterna apps just fine with it when using WSL.

**Comment by avl42** [comment]
If you run your application in WSL, then it is essentially a Linux
application running under some specific "Linux-flavour".

**Comment by mabe02** [comment]
The ANSI escape codes are one thing, but we still rely on stty to some extent for modifying things like echo and special characters. Not sure if there's a way to do that with the window terminal without using native library integration.

**Comment by gonza2323** [comment]
Ah ok, I understand, thank you, The current native integration is for the old console, right?

**Comment by mabe02** [comment]
Yes, it's using JNI and calls the standard system calls (GetConsoleMode, SetConsoleMode, etc).

**Comment by avl42** [comment]
WSL *is* Linux - as far as relevant for lanterna.

The difference between a cmd.exe (that knows ansi-colors)
and a WSL terminal is: access to the features of "stty":
with WSL, you can just call stty,  whereas for plain cmd.exe
you need some native API, thus the special native Terminal-
implementation.

**Comment by avl42** [comment]
point is, that you *absolutely need* the "stty"-stuff for lanterna...
otherwise you couldn't react to single keystrokes.

You certainly can  read a full line from terminal and then output
your response lines without stty, but then you wouldn't need
lanterna in the first place.


Regarding WSL:  I once tried it, but all attempts for WSL 2 failed with
vague errors, only WSL 1 finally

**Comment by avl42** [comment]
sorry, previous message got sent prematurely (Ctrl+Enter pressed, instead
of Shift-Enter)

Regarding WSL:  I once tried it, but all attempts for WSL 2 failed with
vague errors,
only WSL 1 could finally be installed. -> I don't know anything about wsl
2.
